requirements.txt

```diff
@@ -1,4 +1,6 @@
-versioneer
+pypandoc
+pyyaml
 qtpy
 qtpyinheritance
-pyyaml
+requests
+versioneer
```

Comment:
While we're here, does versioneer actually belong in the requirements? It's only used in setup.py.

Reply:
I think it is not required. Let me see if the build passes without it, then we should update our cookiecutter if it's OK.